roles/adoption_osp_deploy/templates/adoption_vars.yaml.j2

```diff
@@ -30,10 +30,12 @@ edpm_nodes:
       ansibleHost: {{ node_nets.networks.ctlplane.ip_v4 }}
     networks:
     {% for net in node_nets.networks.keys() %}
```

Comment:
make sure ocpbm network is not added to networks

Reply:
PR updated, it will now filter on `cifmw_libvirt_manager_pub_net` - i.e ocpbm network.

Reply:
I got error: 'AnsibleUndefinedVariable: ''cifmw_libvirt_manager_pub_net'' is undefined.
cifmw_libvirt_manager_pub_net is not defined during deploy-osp-adoption.yml playbook run, but only during create-infra.yml

Reply:
oh, ok - I changed the approch. Added a default var in adoption_osp_deploy `cifmw_adoption_osp_deploy_adoption_vars_exclude_nets` with value:
` - "{{ cifmw_libvirt_manager_pub_net | default("ocpbm") }}`

I think ci-framework uses `public` as the default for `cifmw_libvirt_manager_pub_net` but I think is is more commonly defined as "ocpbm" ...

Reply:
Thank you for the change. It works for me now.